spark2-sql-plugin/pom.xml

```diff
@@ -0,0 +1,256 @@
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.nvidia</groupId>
+        <artifactId>rapids-4-spark-parent</artifactId>
+        <version>22.02.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>rapids-4-spark-sql-meta_2.11</artifactId>
+    <name>RAPIDS Accelerator for Apache Spark SQL Plugin Base Meta</name>
+    <description>The RAPIDS SQL plugin for Apache Spark Base Meta Information</description>
+    <version>22.02.0-SNAPSHOT</version>
+
+    <properties>
+        <scala.binary.version>2.11</scala.binary.version>
+        <scala.version>2.11.12</scala.version>
+        <spark.version>${spark248.version}</spark.version>
+        <spark.test.version>${spark.version}</spark.test.version>
+        <spark.version.classifier>spark24</spark.version.classifier>
+        <target.classifier>spark24</target.classifier>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>com.google.flatbuffers</groupId>
+            <artifactId>flatbuffers-java</artifactId>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.scala-lang</groupId>
+            <artifactId>scala-library</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.scalatest</groupId>
+            <artifactId>scalatest_${scala.binary.version}</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+          <groupId>org.apache.orc</groupId>
+          <artifactId>orc-core</artifactId>
+          <classifier>${orc.classifier}</classifier>
+          <exclusions>
+            <exclusion>
+              <groupId>org.slf4j</groupId>
+              <artifactId>slf4j-api</artifactId>
+            </exclusion>
+          </exclusions>
+        </dependency>
+        <dependency>
+          <groupId>org.apache.orc</groupId>
+          <artifactId>orc-mapreduce</artifactId>
+          <classifier>${orc.classifier}</classifier>
+          <exclusions>
+            <exclusion>
+              <groupId>com.google.code.findbugs</groupId>
+              <artifactId>jsr305</artifactId>
+            </exclusion>
+          </exclusions>
+        </dependency>
+        <dependency>
+          <groupId>org.apache.hive</groupId>
+          <artifactId>hive-storage-api</artifactId>
+          <exclusions>
+            <exclusion>
+              <groupId>org.slf4j</groupId>
+              <artifactId>slf4j-api</artifactId>
+            </exclusion>
+          </exclusions>
+        </dependency>
```

Comment:
Should we be using the same version as is used by Spark 2.4 here?  If not, don't we need to shade the ORC/Hive classes to avoid interfering with the versions Spark 2.4 is using?  I'm wondering if we really need to explicitly list ORC/Hive here or can just get the transitive dependency via the spark 2.4 jars.  Since we're not going to need to parse/create ORC footers or perform predicate pushdown as part of an explain-only API, I'll be a bit surprised if we really need ORC jars at all.  We may need to use some Hive classes to examine Hive UDFs to see if they are RAPIDS accelerated.

Reply:
yeah we can use the same version I think, 2.4.1-2.4.8 use orc 1.5.5 and 2.4.0 uses orc 1.5.2, I'll check on the hive classes